src/configuration/FirestoreConfigurationContext.tsx

```diff
@@ -239,10 +223,16 @@ export const FirestoreConfigurationContextProvider: React.FC = ({children}) => {
     configurableLinks,
     mobilityOperators,
     harborConnectionOverrides,
+    firestoreConfigStatus,
   ]);
 
   return (
-    <FirestoreConfigurationContext.Provider value={memoizedState}>
+    <FirestoreConfigurationContext.Provider
+      value={{
+        ...memoizedState,
+        resubscribeFirestoreConfig: useCallback(() => subscribeFirestore(), []),
```

Comment:
Same here, needs dependency in dependency list. 

Also, should the previous subscription be unsubscribed before new subscriptions? 

And I think we can be slightly more explicit regarding resubscription:
- Is resubscription only allowed if there is no data? If so it should be a check on that here.
- Or is resubscription also allowed even if there is data? if so the state should cleared and the status should be set to `loading` on resubscription.

Reply:
Good point ☝🏼 Went for the solution with clearing the state and setting to loading.